README.md

```diff
@@ -2,7 +2,7 @@
 
 # React Native CLI Templates
 
-These are project templates that are used by the [Brightlayer UI CLI](https://github.com/etn-ccis/blui-cli) when starting a new Brightlayer UI project. They are used behind the scenes by the Brightlayer UI CLI.
+These are project templates that are used by the React Native CLI when starting a new Brightlayer UI project. They are used behind the scenes by the React Native CLI.
```

Comment:
We can remove this line `They are used behind the scenes by the React Native CLI.`